organizers/models.py

```diff
@@ -5,24 +5,15 @@
 from django.db.models import Avg, F
 from django.utils import timezone
 
+from django.conf import settings
 from applications.models import Application
 from user.models import User
 
+MAX_VOTES = getattr(settings, 'MAX_VOTES')
```

Comment:
nitpick: set default to 10
```suggestion
MAX_VOTES = getattr(settings, 'MAX_VOTES', 10)
```